morf-core/src/test/java/org/alfasoftware/morf/metadata/TestSchemaUtils.java

```diff
@@ -85,8 +86,8 @@ public void testCopySchemaWithExclusions() {
     //...assert during the copy excluded tables and views are being removed
     assertEquals("2 tables should exist", 2, schema.tables().size());
     assertEquals("1 view should exist", 1, schema.views().size());
-    assertFalse(schema.tables().stream().anyMatch(t -> t.getName() == excludePrefix));
-    assertFalse(schema.views().stream().anyMatch(t -> t.getName() == excludeMatch));
+    assertFalse(schema.tables().stream().anyMatch(t -> Objects.equals(t.getName(), excludePrefix)));
+    assertFalse(schema.views().stream().anyMatch(t -> Objects.equals(t.getName(), excludeMatch)));

```

Comment:
Note: `java.util.Objects.equals()` goes further than `==`, and uses `Object.equals()`.
Therefore this is a potential behaviour change and must be investigated.

We are comparing Strings. Objects.equals sounds like a better choice.

Reply:
This is a test where comparing the value rather than the pointer is clearly the intended behaviour.